codice/ddf-ui
# DDF-6008 Return query highlighting results to the UI

This enables the highlighting results from a query to be returned to the UI.

To test:
1) build this DDF PR: https://github.com/codice/ddf/pull/6009 
2) build this PR
3) Start SolrCloud
4) Modify the schema in the catalog collection to store the tokenized text fields (stored="true") 
5) Start DDF and do a standard profile install
6) Add ddf-ui front and back end
7) from the karaf console enable the highlighting capability: `property solr.highlight.enabled true`
8) Ingest data into DDF
9) perform a text query using specific words (i.e. not a wildcard query)
10) verify that the cql response contains the highlighting information.
